### DISCUSSION TITLE
Any plans for the Cases never resolved

### DISCUSSION THREAD

**Discussion by Ignis231** [THGSCST/Vices-and-Sins-MOD#8] [Q&A] [answered]
I see some talk about the cases never reaching a resolution in the discussions, is this something you are working on, or have plans for?

**Comment by THGSCST** [comment, marked as answer]
I couldn't even get to where @locatellidaniel got to. He went far and came very close to solving the bug.

I tried, but my assembly skills are very limited. All my attempts have failed so far. I will only return to this bug if a new tool or plugin emerges that makes reverse engineering easier.

**Reply by Ignis231** [reply]
Shame to hear but I understand, this game does seem to be quite difficult in the backend.
